src/main/kotlin/no/skatteetaten/aurora/boober/utils/ImageStreamTagUtils.kt

```diff
@@ -17,5 +18,17 @@ fun ImageStreamTag.findErrorMessage(): String? {
 
 fun ImageStreamTag.toJsonNode(): JsonNode = jacksonObjectMapper().valueToTree(this)
 
-fun imageStreamTagFromJson(jsonNode: JsonNode?): ImageStreamTag =
-        jacksonObjectMapper().readValue(jsonNode.toString())
+fun imageStreamTagFromJson(jsonNode: JsonNode?): ImageStreamTag {
+
+    if (jsonNode == null) throw IllegalArgumentException("Missing ImageStreamTag response body")
+
+    val image = jsonNode.get("image")
```

Comment:
Can we make a note here that this is because of https://github.com/fabric8io/kubernetes-model/issues/296